CorsixTH/com.corsixth.corsixth.metainfo.xml

```diff
@@ -20,6 +20,127 @@
+          <li>Dutch translation has been updated. Thanks @Alberth289346 &amp;
+              @jetenergy</li>
+        </ul>
+        <p>Bug Fixes</p>
+        <ul>
+          <li>Improved reliability of the Map Editor checking for a path to the
+              hospital on save</li>
+          <li>Game speed tick rates have been smoothed out again, improving
+              animations and sound playback for objects</li>
+          <li>Fixed timing issue with the demo movie playback trigger following
+              game speed changes in 0.68.0</li>
+          <li>Fixed timing issue with random announcements following game speed
+              changes in 0.68.0</li>
+          <li>Improved colour accuracy for fonts in unicode languages now means
+              you can actually read when your game is paused</li>
+          <li>Many fixes for the position of staff and patient moods in the game!
+              More are scheduled to come in future</li>
+          <li>Fixed a bug where the visual mood for dying patients would
+              disappear</li>
+          <li>Fixed an issue with the patient wait mood not clearing as expected</li>
+          <li>Fixed a (very annoying) bug with walks that caused new patients to
+              enter rooms along their path</li>
+          <li>Rework to the operating theatre room to prevent patient/staff
+              lockups</li>
+          <li>Free build now always comes with free heating and free parcel
+              purchases!</li>
+          <li>Heating controls can no longer be adjusted during a boiler
+              breakdown</li>
+          <li>Epidemic timer now ends once all patients are resolved</li>
+          <li>Old serious radiation epidemics can no longer cause a game crash.
+              In future, this epidemic type will be reinstated again</li>
+          <li>Fixed the deactivate epidemic cheat not working as intended</li>
+          <li>Failed cheats will now always provide feedback messages</li>
+          <li>Failed create emergency cheats because you do not have a heliport
+              now error correctly</li>
+          <li>Spawning a contagious patient via cheating now works in levels
+              without a helipad</li>
+          <li>Fixed an issue where the win level cheat may not trigger correctly</li>
+          <li>Fixed a bug where loans were not preventing a lose game scenario
+              for balance</li>
+          <li>The staff salary request window is now properly dismissed if auto
+              grant wage requests is enabled and triggers</li>
+          <li>Fixed an issue where the Staff Management window could interfere
+              with the world</li>
+          <li>Editing a room now closes the machine dialog of the room if it was
+              open</li>
+          <li>Room statistics no longer reset when editing a room</li>
+          <li>Long windows now correctly count as a window</li>
+          <li>Dynamic info for machines now shows during cycling</li>
```

Comment:
Not sure what this means. What is "cycling"?

Reply:
Something about how you can click through the different machine UI (think the title?) will go to the next machine in your hospital.